lib/cfn-nag/custom_rules/password_base_rule_sub_property_with_list.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require 'cfn-nag/util/enforce_reference_parameter'
+require 'cfn-nag/util/enforce_string_or_dynamic_reference'
+require_relative 'base'
+
+class PasswordBaseRuleSubPropertyWithList < BaseRule
```

Comment:
small nit.  when i subclass something, I typically preserve that parent class' name as the suffix, e.g. XBaseRule < BaseRule.

Reply:
I can fix this to be `PasswordSubPropertyWithListBaseRule`